README.md

```diff
@@ -149,20 +223,18 @@ more before you run your local development version of MDN.
 * To run the service:
     * `node run.js`
 * To run tests:
-    * `./node_modules/.bin/mocha tests`
+    * `npm run test`
 * To check code quality:
-    * `./node_modules/.bin/jshint --show-non-errors lib tests`
-        * This will make a racket if it hits `parser.js`
-        * TODO: Ignore this file.
-* To generate document macro parser (optional):
-    * `./node_modules/.bin/pegjs lib/kumascript/parser.pegjs`
-        * This is not required in dev, but should be done for production.
-        * If `parser.js` is missing, the parser will be built on the fly.
+    * `npm run lint`
+* To generate document macro parser (if parser.pegjs is modified):
+    * `make parser.js`
+    * After regenerating this file, add the line `/* eslint-disable */`
+      to the top of the file.
```

Comment:
This rarely changes, and the linter would catch it, but we could automate adding the lint comment:
```
echo "/* eslint-disable */" > src/parser.js
./node_modules/.bin/pegjs -o - src/parser.pegjs >> src/parser.js

Reply:
Thanks for that suggestion. Done in commit  682f6bd